src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -170,6 +170,8 @@
     <Compile Include="System\Resources\ResourceSet.cs" />
     <Compile Include="System\Runtime\InteropServices\CriticalHandle.cs" />
     <Compile Include="System\Text\CodePageDataItem.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.Unix.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.HillClimbing.Unix.cs" />
```

Comment:
Is there anything Unix-specific in the file? Maybe call it just `ClrThreadPool.HillClimbing.cs` ?